tests/integration/test_orfs_flows.py

```
@@ -0,0 +1,573 @@
+
+
+# ---------------------------------------------------------------------------
+# Test: ORFS filesystem integration (requires real ORFS installation)
+# ---------------------------------------------------------------------------
+
+
+@skip_if_no_orfs
+class TestORFSFilesystemIntegration:
+    """Tests that use the real ORFS flow directory structure."""
+
+    async def test_orfs_platforms_discoverable(self, mcp_client):
```

Ruff (0.15.2)

[warning] Unused method argument: `mcp_client`

(ARG002)